src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -196,9 +196,11 @@ def prepare_problem_for_grouping(
 
 
 def fingerprint_group(transaction_name, span_op, hash, problem_class):
-    signature = (str(transaction_name) + str(span_op) + str(hash)).encode("utf-8")
+    signature = (str(transaction_name) + str(span_op) + str(hash) + str(problem_class)).encode(
+        "utf-8"
+    )
     full_fingerprint = hashlib.sha1(signature).hexdigest()
-    return f"1-{problem_class}-{full_fingerprint}"
+    return full_fingerprint
```

Comment:
@udameli Not sure if i messed up the rebase here, is this code correct?

Reply:
yeah it should be `f"1-{problem_class}-{full_fingerprint}"`. I pushed the fix 👍

Reply:
re-pushed. should be fixed now